cmd/draino/draino.go

```diff
@@ -172,8 +173,18 @@ func main() {
 	}
 
 	cf := cache.FilteringResourceEventHandler{FilterFunc: kubernetes.NewNodeConditionFilter(*conditions), Handler: h}
-	lf := cache.FilteringResourceEventHandler{FilterFunc: kubernetes.NewNodeLabelFilter(*nodeLabels), Handler: cf}
-	nodes := kubernetes.NewNodeWatch(cs, lf)
+
+	var nodeLabelFilter cache.ResourceEventHandler
+
+	log.Sugar().Infof("label expression: %v", nodeLabelsExpr)
+	nodeLabelFilterFunc, err := kubernetes.NewNodeLabelFilter(nodeLabelsExpr)
```

Comment:
It feels like we could retain nodeLabels by e.g. using the expression builder to convert the old-style syntax to the new. I hacked together a little demo: https://play.golang.org/p/rgS2rPm1SEP